### ISSUE TITLE
filter with segmented button does not work on mobile

### ISSUE THREAD

**Issue by marianfoo** [marianfoo/ui5-lib-diff#2]
1. start app
2. choose two versions to compare
3. trigger filter for `FIX`
4. error happens

Line:
https://github.com/SAP/openui5/blob/d1372e3eaae237e646432663ac842c1e8fcb5f22/src/sap.uxap/src/sap/uxap/ObjectPageLayout.js#L2622

```js
// BCP 1680331690. Should skip subsections that are in a section with lower importance, which makes them hidden.
var sectionParent = this.oCore.byId(oSectionBase.getId()).getParent();
if (sectionParent instanceof ObjectPageSection && sectionParent._getIsHidden()) {
    return;
}
```

There are sevarell sections, but here the section is not found.

```bash
ObjectPageLayout.js:2492 Uncaught TypeError: Cannot read properties of undefined (reading 'getParent')
    at fnClass.<anonymous> (ObjectPageLayout.js:2492:62)
    at Array.forEach (<anonymous>)
    at ObjectPageLayout._updateScreenHeightSectionBasesAndSpacer (ObjectPageLayout.js:2445:23)
    at constructor.reSchedule (ThrottledTaskHelper.js:51:33)
    at ObjectPageLayout._requestAdjustLayout (ObjectPageLayout.js:1892:28)
    at ObjectPageLayout.triggerPendingLayoutUpdates (ObjectPageLayout.js:2930:9)
    at ObjectPageLayout._storeScrollLocation (ObjectPageLayout.js:4245:8)
    at ObjectPageLayout.onBeforeRendering (ObjectPageLayout.js:711:8)
    at Element._handleEvent (Element.js?eval:343:22)
    at triggerBeforeRendering (RenderManager.js?eval:834:14)
```

**Comment by marianfoo** [comment]
https://github.com/SAP/openui5/issues/3828

**Comment by marianfoo** [comment]
bug is fixed, but it seems not shipped with `1.119`, have to wait for `1.120`
